pubspec.yaml

```diff
@@ -4,8 +4,8 @@ version: 0.3.0
 homepage: https://github.com/letsar/value_layout_builder
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.24.0-10.2.pre"
+  sdk: '>=2.12.0 <3.0.0'
+  flutter: '>=2.0.0 <3.0.0'
```

Comment:
If we can keep the lower bound of Flutter it would be better, so that people could still use this version even if they did migrated to 2.0

Reply:
Changed back :)